deploy/cr.yaml

```diff
@@ -273,3 +273,16 @@ spec:
 #          max_worker_processes: 2
 #          shared_buffers: 1GB
 #          work_mem: 2MB
+
+#  customExtensions:
```

Comment:
Wdyt about calling this simply `extension`, because they are just PG extensions, nothing custom about them, it can be, but probably it is something off the shelf, like citus or something.

Reply:
I'm ok!

Reply:
I'm OK to do it but for the record I don't like this:

```
extension:
  extensions:
  - name: pg_cron
     version: 1.6.1
```